SEED-platform/seed
# Fix duplicate users per organization

#### What's this PR do?
Fixes the condition where a user can be added to an organization more than once

#### How should this be manually tested?
1. Before running this migration, follow the steps to reproduce in the linked issue
2. Run the migration, ensure that the `orgs_organizationuser` table has been cleaned up

#### What are the relevant tickets?
#4032
